WebHostLib/customserver.py

```diff
@@ -39,7 +39,7 @@ def _cmd_video(self, platform, user):
 import MultiServer
 
 MultiServer.client_message_processor = CustomClientMessageProcessor
-del (MultiServer)
+del MultiServer
```

Comment:
this looks awkward, i think something like
```python
def inject_custom_message_processor():
    import MultiServer
    MultiServer.client_message_processor = CustomClientMessageProcessor

inject_custom_message_processor()
````
would likely be easier to understand/read?

Reply:
I think that would lose explicitness, as you're relying on the import falling out of scope.

Reply:
I'm pretty sure imports are cached anyway and the del does little to nothing though, just cleans up namespace

Reply:
If it's pure python you can actually do that:
```python
>>> import MultiServer, sys
>>> sys.getrefcount(MultiServer)
3
>>> from MultiServer import Context
>>> sys.getrefcount(MultiServer)
3
>>> MultiServer.client_message_processor = 'a'
>>> del MultiServer
>>> del sys.modules["MultiServer"]
>>> import MultiServer
>>> MultiServer.client_message_processor
<class 'MultiServer.ClientMessageProcessor'>
```
for C modules ymmv.